test/Microsoft.AspNet.Mvc.Razor.Test/Compilation/CompilerCacheTest.cs

```diff
@@ -103,9 +103,8 @@ private static Stream GetMemoryStream(string content)
         }
 
         [Theory]
-        [InlineData(10000)]
+        [InlineData(10000)] // expected failure: same time and length (PranavK)
```

Comment:
:information_source: the duplicate test changes are in a separate commit.  I'll move the sample dependency cleanup into its own commit as well.